src/qmapshack/poi/CPoiPOI.cpp

```diff
@@ -126,6 +127,45 @@ void CPoiPOI::draw(IDrawContext::buffer_t& buf)
     mutex.unlock();
 }
 
+bool CPoiPOI::findPoiCloseBy(const QPoint &px, poi_t & poiItem) const
+{
+    QPointF p = px;
+    poi->convertPx2Rad(p);
+    for(const QString& category : categoryActivated.keys(Qt::Checked))
+    {
+        int minLonM10 = qFloor(p.x() * RAD_TO_DEG * 10);
+        int minLatM10 = qFloor(p.y() * RAD_TO_DEG * 10);
+        //The tile that contains the POI has to be loaded, since a position on screen is given
+        for(const rawPoi_t& poiItemFound : loadedPOIs[category][minLonM10][minLatM10])
+        {
+            QPointF pt = poiItemFound.coordinates;
+            poi->convertRad2Px(pt);
+            QPointF x = px - pt;
+            if(x.manhattanLength() < 10)
+            {
+                poiItem.pos = poiItemFound.coordinates;
+                for(auto tag : {"name=", "brand=", "operator="})
+                {
+                    const QStringList& matches = poiItemFound.data.filter(tag);
+                    if (!matches.isEmpty())
+                    {
+                        poiItem.name = matches[0];
+                        poiItem.name.replace(tag, "");
+                        break;
+                    }
+                }
+                if(poiItem.name == "")
```

Comment:
`isEmpty()` is always preferred over comparison with a null string. I would even expect it to be more performant.